frontend/src/pages/ActivityReport/Pages/Review/index.js

```diff
@@ -87,7 +85,10 @@ const ReviewSubmit = ({
             error={error}
             onSaveForm={onSaveForm}
           >
-            <Accordion bordered={false} items={items} />
+            <>
```

Comment:
What does the "<>" or "</>" do?

Reply:
It just a short hand way of writting Fragment <Framgment/>.